src/main/resources/application.properties

```diff
@@ -68,6 +68,14 @@ spring.security.oauth2.client.registration.oidc.client-id = internal
 spring.security.oauth2.client.registration.oidc.client-secret = internal
 oidc.issuer = ${OIDC_ISSUER:http://fr-am:8080/openam/oauth2/hmcts}
 
+idam.client.id=ccd_data_store_api
+idam.client.secret=${IDAM_OAUTH2_DATA_STORE_CLIENT_SECRET:idam_data_store_client_secret}
+idam.client.scope=profile openid roles manage-user
+idam.client.redirect_uri=http://ccd-data-store-api/oauth2redirect
+
+idam.data.store.system.user.username=${IDAM_DATA_STORE_SYSTEM_USER_USERNAME:data.store.idam.system.user@gmail.com}
+idam.data.store.system.user.password=${IDAM_DATA_STORE_SYSTEM_USER_PASSWORD:Pa55word11}
```

Comment:
These are now introduced in addition to the correct ones on lines 90-91 (with a minor difference of `system.user` vs `system-user`? Delete the incorrect ones.